templates/components/entry.html.twig

```diff
@@ -49,9 +49,9 @@
                         </h2>
                     {% endif %}
                 {% elseif(entry.visibility is same as 'trashed') %}
-                    <p class="text-muted">{{ 'deleted_by_moderator' }}</p>
+                    <p class="text-muted"><i>{{ 'deleted_by_moderator'|trans }}</i></p>
```

Comment:
Not sure on `<i>` tags here, I thought `<i>` and `<b>` had both been deprecated out of html, but it looks like https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em#i_vs._em `<i>` can still be used for "technical terms" but it's hard to say what that exactly means, perhaps if this is an error code or something, it would count

Reply:
I didn't like the text without italic so I used: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i